docker/ci/Dockerfile.local

```diff
@@ -17,11 +17,11 @@ ARG VERSION="latest"
 FROM veracruz/ci-base:${VERSION} as final
 
 ARG ARCH=x86_64
-ARG ICECAP_RUST_VERSION=1.60.0
```

Comment:
Rust 1.60.0 is only used for IceCap. So you should probably remove all commands using `ICECAP_RUST_VERSION`.